packages/router/__tests__/fetchers-test.ts

```diff
@@ -122,7 +122,10 @@ describe("fetchers", () => {
     });
 
     it("loader fetch", async () => {
-      let t = initializeTest({ url: "/foo" });
+      let t = initializeTest({
+        url: "/foo",
+        hydrationData: { loaderData: { root: "ROOT", foo: "FOO" } },
+      });
```

Comment:
Same as above - these tests can be ignored as they just sure up the `hydrationData` for existing tests